timescaledb/tutorials/nfl-fantasy-league.md

```diff
@@ -0,0 +1,188 @@
+# Win your NFL fantasy league with TimescaleDB
+
+This tutorial is a step-by-step guide on how to ingest and analyze american football data with TimescaleDB.
+
+The dataset that we're using is provided be the National Football League (NFL) and contains data about 
+all the passing plays of the 2018 NFL season. We're ingesting this dataset and exploring it in fun ways. You will also see how this dataset and using TimescaleDB can improve your NFL fantasy league strategy. If you aren't an NFL fan, don't worry, you will still find this tutorial helpful as it's showcasing how to get started with TimescaleDB and explore a real world dataset with SQL and Python.
+
+1. [Create tables][]
```

Comment:
should we add prerequisites and about the data set in the list? It feels a little confusing to have this list then have to wait to see it manifest.

Reply:
https://github.com/timescale/docs/pull/187/commits/25196ee59ae89b060d067d45367bbbe51b4aaa90